### ISSUE TITLE
Skia - ListView scrollbar initially not shown

### ISSUE THREAD

**Issue by sarosigy** [unoplatform/uno#8939]
### Current behavior

On linux platform, when filling a ListView with a lot of items, the scrollbar is not shown, not even at hovering over the edge of the ListView.
After resizing the application window (even just a tiny bit) the scrollbar appears.
![Screenshot_lvscrl1](https://user-images.githubusercontent.com/37872626/171141469-2581923c-18e2-4ad2-8078-3cd97ad2dc9f.png)
![Screenshot_lvscrl2](https://user-images.githubusercontent.com/37872626/171141485-f6e45ba5-0780-40bc-bd36-3de4980d1974.png)
![Screenshot_lvscrl3](https://user-images.githubusercontent.com/37872626/171141504-56873b9f-a9c3-4a2d-971f-9c3893e45271.png)

### Expected behavior

The scrollbar should be shown when the number of items would require.

### How to reproduce it (as minimally and precisely as possible)

Full source at link:

https://github.com/sarosigy/uno-lvtest/

### Workaround

Apart from the application window resize, don't know

### Works on UWP/WinUI

Yes

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### NuGet package version(s)

Uno.WinUI.Skia.Gtk Version="4.3.6"

### Affected platforms

Skia (GTK on Linux/macOS/Windows)

### IDE

Visual Studio Code

### IDE version

1.67.2

### Relevant plugins

Uno Platform 0.4.0

**Comment by ramezgerges** [comment]
@sarosigy Could you test it again using the latest version? I tried this on Skia.Gtk (although on Windows) and it doesn't seem to reproduce.

**Comment by sarosigy** [comment]
The 4.9.42 also has this bug, the scrollbar still only shows after a resize.

**Comment by ramezgerges** [comment]
Indeed. It seems to be fixed on the latest 5x build, but it hasn't been backported to 4x.

**Comment by sarosigy** [comment]
Now I tested with 5.0.0-dev.1684 and the issue still persists.

**Comment by MartinZikmund** [comment]
# Repro generator

Runtime test(s) added in branch [`dev/mazi/repro/8939`](https://github.com/unoplatform/uno/compare/dev/mazi/repro/8939):
- `src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs` → `When_ManyItems_Then_ScrollBarShownInitially`

⚠️ The test(s) **pass** on current master — the ListView's ComputedVerticalScrollBarVisibility is Visible and ScrollableHeight is > 0 for 100 items in a 300px tall ListView on Skia. The bug may have been fixed since the original report in 2022. The test verifies the ScrollViewer state programmatically; the original issue may have been about visual rendering specifically on Skia/GTK where the ScrollBar indicator wasn't drawn despite correct logical state.
